.github/workflows/integration-wf.yml

```diff
@@ -0,0 +1,85 @@
+name: Integration Test Suite WorkFlow
+
+on: 
+  push:
+    branches:
+      - integration_test
+  pull_request:
+    branches:
+      - master
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the current repo
+        uses: actions/checkout@v1
+      - name: Install Tool
+        run: |
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-linux-amd64
+          chmod +x ./kind
+          sudo mv ./kind /usr/bin/
+      - name: Run Kind Cluster
+        run: |
+          cd integration/
+          ./registry.sh
+          helm repo add kedacore https://kedacore.github.io/charts
+          helm repo update
+          kubectl create namespace keda
+          helm install keda kedacore/keda --namespace keda
+      - name: Create Docker Image for HTTP server
+        run: |
+          cd integration/server/
+          docker build -t localhost:5000/http-server:latest .
+          docker push localhost:5000/http-server:latest
+      - name: Create Docker Image for KEDA Connector
+        run: |
```

Comment:
Can we logically group these tasks - for ex. all image building tasks first, then all deployment to K8S etc.? Just easier to read

Reply:
I think so that's how we have right now, once the Kind cluster with local registry is up, we create docker images and push to local registry to be used in deployments which are deployed afterwards.